src/main/kotlin/id/walt/services/essif/EssifClient.kt

```diff
@@ -18,13 +18,13 @@ import id.walt.model.DidMethod
 import id.walt.model.OidcRequest
 import id.walt.services.did.DidService
 import id.walt.services.jwt.keyId
-import id.walt.services.vc.VCService
+import id.walt.services.vc.JsonLdCredentialService
 
 
 object EssifClient {
 
     private val log = KotlinLogging.logger {}
-    private val credentialService = VCService.getService()
+    private val credentialService = JsonLdCredentialService.getService()
```

Comment:
I think the implementation to use should be configurable. Furthermore, ESSIF/EBSI only supports JWT Proof for now